MonoZelda/Scenes/RoomScene.cs

```diff
@@ -56,13 +53,16 @@ public override void LoadContent(ContentManager contentManager)
         // Need to wait for LoadContent because MonoZeldaGame is going to clear everything before calling this.
         LoadRoom(contentManager);
 
-        // Play dungeon theme
-        SoundManager.PlaySound("LOZ_Dungeon_Theme",true);
+        // create player sprite classes
+        playerSprite = new PlayerSpriteManager();
+        var playerSpriteDict = new SpriteDict(SpriteType.Player, SpriteLayer.Player, PlayerState.Position);
+        playerSprite.SetPlayerSpriteDict(playerSpriteDict);
 
-        playerSprite = new PlayerSpriteManager(playerState);
-        var takeDamageCommand = new PlayerTakeDamageCommand(playerState, playerSprite);
+        // Play Dungeon Theme
+        SoundManager.PlaySound("LOZ_Dungeon_Theme", true);
```

Comment:
Need to create a static Sounds class to centralize all of the sound name strings.